### PR TITLE
Added a badge filter for the UI/UX'er.

### LINKED ISSUE #3379: [UX] Design: Need a new community badge for UI/UX contributors
**Current Behavior**

We need a new batch to recognize and appreciate the work of UI/UX'ers in our community. 
image

**Desired Situation**

---
[Optional] **Alternatives**

[Optional] **Additional context**

Badges are provided to consistent contributors and people who are active in the community, who hold a good amount of knowledge about the project and the repository, and who are associated to that project for a long time, and are the best people to reach out to. We have different badges for different projects; Landscape, Meshery, SMP etc. You can find different badges for different repositories Layer5 has to offer, in the members section on the layer5 site.
https://layer5.io/community/members

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sam1code**
Signed-off-by: Soumyakanta Pattanaik <samcse95@gmail.com>

**Description**

This PR fixes #3379

**Notes for Reviewers**
Added another badge for uiux'er

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 942f0068243c3a00778653924cb3d5be712c9ff1 at: https://6350f6be0e13317e77d2a745--layer5.netlify.app

**Comment by sam1code** [comment]
yes Only filter addeed... Once the badge is available need to attach it.

**Comment by leecalcote** [comment]
@sam1code, @DivineOrji now has a final version the icon for this badge. Please connect with him in order to add to this PR.

**Comment by sam1code** [comment]
Thankyou @leecalcote

**Comment by leecalcote** [comment]
Ping when the SVG version is up.

**Comment by leecalcote** [comment]
@sam1code @DivineOrji has the new version of the logo been added here?

**Comment by deepeshaburse** [comment]
@sam1code @DivineOrji The logo hasn't been added yet, kindly add it.

**Comment by deepeshaburse** [comment]
> Im really sorry @deepeshaburse after a small accident I stopped my work for some time.
> Just give me some hours I'm creating the PR

I'm so sorry to hear that. Take care. Thanks for the update!

**Comment by leecalcote** [comment]
@sam1code @DivineOrji ... this week or bust.

**Comment by Anita-ihuman** [comment]
Hey Sam, @sam1code whats the progress on this PR?

**Comment by sam1code** [comment]
Really sorry to say that, I lost my laptop on a accident and I tried to do with another laptop but there was some issue but I'm sure that Today I'll make the final PR as I refurbished the PC and also now I'm again able to work.

**Comment by leecalcote** [comment]
@sam1code, ok. Please execute (last chance).

**Comment by sam1code** [comment]
All set..

**Comment by l5io** [comment]
🚀 Preview for commit 67faa8da9ba7117f8dac8b116275f9f96c4ec564 at: https://63a4e241d46c0230bd031107--layer5.netlify.app

**Review by eeshaanSA — COMMENTED** [review]
UI|UX 
Is this better than UI/UX'er?
And if we are going with UI/UX'er, wouldn't UI/UX'ers make more sense? @deepeshaburse @leecalcote

File: src/sections/Community/Member-single/index.js [hunk 1]
```diff
@@ -156,6 +157,17 @@ const MemberSingle = ({ frontmatter }) => {
...
                       )}
+                      {badges.includes("patterns") && (
+                        <li>
+                          <Link to="/learn/service-mesh-books/service-mesh-patterns">
+                            <img
+                              className="profile-social-links"
+                              src={uiuxrLogo}
+                              alt="UI/UX'er Logo"
+                            />
+                          </Link>
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Member-single/index.js:168]:
```suggestion
                          <img
                            className="profile-social-links"
                            src={uiuxrLogo}
                            alt="UI/UX'er Logo"
                          />
```

**Review by Nikhil-Ladha — APPROVED** [review]
Let's take it in.

**Pull request merged**